platform/mellanox/docker-syncd-mlnx.mk

```diff
@@ -1,19 +1,4 @@
 # docker image for mlnx syncd
 
-DOCKER_SYNCD_MLNX = docker-syncd-mlnx.gz
-$(DOCKER_SYNCD_MLNX)_PATH = $(PLATFORM_PATH)/docker-syncd-mlnx
-$(DOCKER_SYNCD_MLNX)_DEPENDS += $(SYNCD) $(PYTHON_SDK_API)
-$(DOCKER_SYNCD_MLNX)_PYTHON_DEBS += $(MLNX_SFPD)
-$(DOCKER_SYNCD_MLNX)_LOAD_DOCKERS += $(DOCKER_CONFIG_ENGINE_STRETCH)
-SONIC_DOCKER_IMAGES += $(DOCKER_SYNCD_MLNX)
-SONIC_STRETCH_DOCKERS += $(DOCKER_SYNCD_MLNX)
-ifneq ($(ENABLE_SYNCD_RPC),y)
-SONIC_INSTALL_DOCKER_IMAGES += $(DOCKER_SYNCD_MLNX)
-endif
-
-$(DOCKER_SYNCD_MLNX)_CONTAINER_NAME = syncd
-$(DOCKER_SYNCD_MLNX)_RUN_OPT += --net=host --privileged -t
-$(DOCKER_SYNCD_MLNX)_RUN_OPT += -v /host/machine.conf:/etc/machine.conf
-$(DOCKER_SYNCD_MLNX)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
-$(DOCKER_SYNCD_MLNX)_RUN_OPT += -v /host/warmboot:/var/warmboot
-
+DOCKER_SYNCD_PLATFORM_CODE = mlnx
+include $(PLATFORM_PATH)/../template/docker-syncd-base.mk
```

Comment:
can you check if docker-syncd-bfn.mk can use this base? syncd docker seems different for different asic, not sure if all of them can use same template.

Reply:
Moved the DEPENDS to the individual make files and let the common stuff alone stay in template.

The only conflict would be that VS, adds self to SONIC_INSTALL_DOCKER_IMAGES, irrespective of ENABLE_SYNCD_RPC flag value, where as the rest add only if flag is not set to 'y'.

This can be easily fixed in docker-syncd-vs.mk, by adding it, if flag set to 'y'.